WcaOnRails/config/locales/ru.yml

```diff
@@ -1921,7 +1998,7 @@ ru:
       not_confirmed_not_visible: Соревнование не подтверждено и невидимо широким массам
       #original_hash: a821396
       stripe_connected: >-
-        Вы успешно подключили ваш Stripe аккаунт и теперь можете получать
+        Вы успешно подключили ваш Stripe аккаунт и теперь можете получать 
```

Comment:
Travis is complaining about this trailing whitespace :slightly_frowning_face: I'm going to close this PR, please re-submit it when you can @Claster.